apps/mobile-alphabet/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "mobile-alphabet",
+    "version": "0.0.1",
```

Comment:
Be aware that we'll want to bump this with releases as part of the release cycle. There may be tooling to automate this with Nx. We can think about this when the time comes.